lib/p2p/Pool.ts

```diff
@@ -65,10 +65,18 @@ class Pool extends EventEmitter {
     }
 
     const peer = Peer.fromOutbound(socketAddress);
-    await this.connectPeer(peer);
+    await this.tryConnectPeer(peer);
     return peer;
   }
 
+  private tryConnectPeer = async (peer: Peer): Promise<void> => {
+    try {
+      await this.connectPeer(peer);
+    } catch (err) {
+      this.logger.warn(`connectPeer failed: ${err}`);
```

Comment:
I think `warn` might be too "urgent" of a logger level here. To me at least, a warning is cause for concern, whereas failing to connect on startup is is something that will happen with some regularity. I was thinking `verbose` might be the most appropriate, but maybe `info` is fine if we want to log failed connections on startup as a matter of routine.

Reply:
I think that because `connect` returns the connection status, there’s no need to be dependent on the logging there, and changing it to `info` should be fine.